### PR TITLE
Update section.md

### PR DESCRIPTION
**Pull request by D-Salkovic**
I added some changes that were proposed [here](https://github.com/Codecademy/docs/pull/1227#issuecomment-1307470608) and I am looking forward for your feedback.

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/html/concepts/semantic-html/terms/section/section.md
Comment (SSwiniarski) [inline content/html/concepts/semantic-html/terms/section/section.md:39]: These examples belong better in their own example section after the existing example section, rather than in the syntax section. Perhaps with the header "Alternatives to `<section>`"

File: content/html/concepts/semantic-html/terms/section/section.md
Comment (SSwiniarski) [inline content/html/concepts/semantic-html/terms/section/section.md:83]: Make sure there's a line between paragraphs and the code blocks.

File: content/html/concepts/semantic-html/terms/section/section.md
Comment (SSwiniarski) [inline content/html/concepts/semantic-html/terms/section/section.md:41]: Because of the interspersed code blocks, this doesn't really work as a bulleted list. I'd remove the bullets and have the code blocks introduced with regular paragraphs.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
I did a first review and left a few comments.

**Comment by D-Salkovic** [comment]
I resolved the issues you mentioned and I also created a new header and changed the order so that it goes as following: _Sythax_, _Examples_, _Alternatives to `<section>`_.
:)

**Review by SSwiniarski — APPROVED** [review]
@D-Salkovic Looks good for a second review.

**Comment by D-Salkovic** [comment]
What is my next step to finishing this PR.
I am new to learning the GitHub system :)

**Comment by SSwiniarski** [comment]
@D-Salkovic, One of the other maintainers will come and do a second review and probably have their own comments for you to address. After that's complete we'll merge the PR.

File: content/html/concepts/semantic-html/terms/section/section.md
Comment (yangc95) [inline content/html/concepts/semantic-html/terms/section/section.md:16]:
```suggestion
The **`<section>`** HTML element defines elements in a document, such as chapters, headings, or any other area of the document with the same theme.
```

File: content/html/concepts/semantic-html/terms/section/section.md
Comment (yangc95) [inline content/html/concepts/semantic-html/terms/section/section.md:68]:
```suggestion
There are also examples where we can use other elements that semantically fit better than `<section>` element.
```

File: content/html/concepts/semantic-html/terms/section/section.md
Comment (yangc95) [inline content/html/concepts/semantic-html/terms/section/section.md:89]:
```suggestion
The second example is the `<nav>` element. It is used to describe that the content inside is a navigation system of the page.
```

File: content/html/concepts/semantic-html/terms/section/section.md
Comment (yangc95) [inline content/html/concepts/semantic-html/terms/section/section.md:96]:
```suggestion
    <!-- Navigation of the page -->
```

File: content/html/concepts/semantic-html/terms/section/section.md
Comment (yangc95) [inline content/html/concepts/semantic-html/terms/section/section.md:24]:
```suggestion
  <!-- Elements in a document with a common theme -->
```

**Review by yangc95 — COMMENTED** [review]
@D-Salkovic I left some suggestions

**Comment by D-Salkovic** [comment]
@yangc95 I resolved the suggestions you made. Hopefully everything looks good now :)

File: content/html/concepts/semantic-html/terms/section/section.md
Comment (yangc95) [inline content/html/concepts/semantic-html/terms/section/section.md:66]: @D-Salkovic upon second look, we should turn these into bullets (without the html code block that follows) and link out to the respective entries on each element like:


There are also examples where we can use other elements that semantically fit better than the `<section>` element:
  
- The [`<header>`](https://www.codecademy.com/resources/docs/html/semantic-html/header) element which can be used to describe that the content inside is the header of the page.
- The [`<nav>`](https://www.codecademy.com/resources/docs/html/semantic-html/nav) element. It is used to describe that the content inside is a navigation system of the page.
- The [`<main>`](https://www.codecademy.com/resources/docs/html/semantic-html/main) element which is used to describe that the content inside is the main content of the page.
- The [`<footer>`](https://www.codecademy.com/resources/docs/html/semantic-html/footer) element which is used to describe that the content inside is the footer of the page.

**Comment by yangc95** [comment]
@D-Salkovic I attempted to resolve some of the failing tests but was unable to due to your branch's permissions so when you get a chance, make sure to pass the checks! I also left one comment at the end about alternatives 🙂

File: content/html/concepts/semantic-html/terms/section/section.md
Comment (yangc95) [inline content/html/concepts/semantic-html/terms/section/section.md:68]:
```suggestion
There are also examples where we can use other elements that semantically fit better than the `<section>` element.
```

**Comment by D-Salkovic** [comment]
> @D-Salkovic I attempted to resolve some of the failing tests but was unable to due to your branch's permissions so when you get a chance, make sure to pass the checks! I also left one comment at the end under alternatives 🙂

@yangc95 I have made the changes you proposed.
I am sorry about the permissions, but i kinda do not understand, im a beginner so if you could guide me or give me some helpful resources on how to resolve the problem. :) 
Thanks in advance.

File: content/html/concepts/semantic-html/terms/section/section.md
Comment (Dusch4593) [inline content/html/concepts/semantic-html/terms/section/section.md:66]:
```suggestion
## Alternative Elements
```

File: content/html/concepts/semantic-html/terms/section/section.md
Comment (Dusch4593) [inline content/html/concepts/semantic-html/terms/section/section.md:68]:
```suggestion
The `<section>` element should only be used if no other suitable element is available. Some alternatives include the following:
```

File: content/html/concepts/semantic-html/terms/section/section.md
Comment (Dusch4593) [inline content/html/concepts/semantic-html/terms/section/section.md:88]:
```suggestion
```

File: content/html/concepts/semantic-html/terms/section/section.md
Comment (Dusch4593) [inline content/html/concepts/semantic-html/terms/section/section.md:111]:
```suggestion
```

File: content/html/concepts/semantic-html/terms/section/section.md
Comment (Dusch4593) [inline content/html/concepts/semantic-html/terms/section/section.md:132]:
```suggestion
```

File: content/html/concepts/semantic-html/terms/section/section.md
Comment (Dusch4593) [inline content/html/concepts/semantic-html/terms/section/section.md:158]:
```suggestion
```

File: content/html/concepts/semantic-html/terms/section/section.md
Comment (Dusch4593) [inline content/html/concepts/semantic-html/terms/section/section.md:133]:
```suggestion
- The [`<footer>`](https://www.codecademy.com/resources/docs/html/semantic-html/footer) element describes that the content inside is the footer of the page.
```

File: content/html/concepts/semantic-html/terms/section/section.md
Comment (Dusch4593) [inline content/html/concepts/semantic-html/terms/section/section.md:112]:
```suggestion
- The [`<main>`](https://www.codecademy.com/resources/docs/html/semantic-html/main) describes that the content inside is the main content of the page.
```

File: content/html/concepts/semantic-html/terms/section/section.md
Comment (Dusch4593) [inline content/html/concepts/semantic-html/terms/section/section.md:89]:
```suggestion
- The [`<nav>`](https://www.codecademy.com/resources/docs/html/semantic-html/nav) element describes that the content inside is a navigation system of the page.
```

File: content/html/concepts/semantic-html/terms/section/section.md
Comment (Dusch4593) [inline content/html/concepts/semantic-html/terms/section/section.md:70]:
```suggestion
- The [`<header>`](https://www.codecademy.com/resources/docs/html/semantic-html/header) element describes that the content inside is the header of the page.
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@D-Salkovic I fixed the failing tests in 333bd8b5881e5223423aa3b27edeb55e03f55230 (hopefully) but also had some comments/suggestions 😄

File: content/html/concepts/semantic-html/terms/section/section.md
Comment (yangc95) [inline content/html/concepts/semantic-html/terms/section/section.md:24]:
```suggestion
  Elements in a document with a common theme
```

File: content/html/concepts/semantic-html/terms/section/section.md
Comment (yangc95) [inline content/html/concepts/semantic-html/terms/section/section.md:16]:
```suggestion
The **`<section>`** element defines elements in a document, such as chapters, headings, or any other area of the document with a common theme.
```

**Review by yangc95 — APPROVED** [review]
LGTM

**Review by Dusch4593 — APPROVED** [review]
@D-Salkovic LGTM! 👍🏻

**Comment by Dusch4593** [comment]
Congrats and thank you for contributing to Codecademy Docs, @D-Salkovic! 🎊 🎉 🥳

Your changes should now be live: https://www.codecademy.com/resources/docs/html/semantic-html/section

Currently, your profile is anonymous on the entry page. To make it visible, you'll need to add your GitHub username to your Codecademy profile, log out of Codecademy, and log back in through your GitHub account:

Screen Shot 2022-12-20 at 10 02 33 AM

Looking forward to your next Docs contribution! 😄

**Pull request merged**
